storage/aptosdb/src/pruner/ledger_store/ledger_store_pruner.rs

```diff
@@ -13,9 +23,12 @@ pub const LEDGER_STORE_PRUNER_NAME: &str = "ledger store pruner";
 pub struct LedgerStorePruner {
```

Comment:
It's not for the "ledger store" per se, so let's rename.

Reply:
Yes, I thought about a name but couldn't think of a something suitable :). Any suggestion?

Reply:
`LedgerPruner`